packages/react-liveness/src/components/FaceLivenessDetector/shared/FaceLivenessErrorModal.tsx

```diff
@@ -116,7 +121,7 @@ export const FaceLivenessErrorModal: React.FC<FaceLivenessErrorModalProps> = (
 
   return (
     <Overlay className={LivenessClassNames.OpaqueOverlay}>
-      <Toast>
+      <Toast ariaLabelledBy="error-heading" role="dialog">
```

Comment:
Can you verify what this experience is with a screen reader? I think the intent is to expose this error message when it appears to the user so I would run through it with VoiceOver (really we should be doing that any time we're adding roles and adding ARIA attributes). I think we'll need a few changes for that to happen:

- This should probably be an `alertdialog` since it's preventative of the experience proceeding and has important information the user needs to know about the state of the UI. There is an example in the [APG here](https://www.w3.org/WAI/ARIA/apg/patterns/alertdialog/examples/alertdialog/). I also like that they include the additional error text in aria-describedby. That means when the dialog appears it announce the title, the additional text in the dialog, and then the focusable element (in their case, the "no" button, in our case the "Try again" button). I think you'll want to make sure those elements always exist, though; I don't know if every error message has additional text.
- We'll need to move focus to the dialog when it's added to the DOM. In that example, they focus the button which I think would work for us, too.

So if you run through this in VoiceOver, I would expect if you fail the challenge (like ducking out of view of it):
- The dialog appears
- The "Try again" button is focused,
- VoiceOver announces "Time out" (or w/e the title is), followed by the additional text "Face didn't fit inside oval in time limit. Try again and completely fill the oval with face in it." (or w/e it is), then the text of the button "Try again"

Reply:
Added the `aria-describedby` property and tested with VoiceOver. Button focuses and VoiceOver announces the heading and dialog description then the text of the button.